vllm/lora/punica_wrapper/punica_cpu.py

```diff
@@ -0,0 +1,345 @@
+from typing import Callable, Optional, Tuple, Union, final
+
+import torch
+
+from vllm.lora.ops.torch_ops.lora_ops import (bgmv_expand, bgmv_expand_slice,
+                                              bgmv_shrink, sgmv_expand,
+                                              sgmv_expand_slice, sgmv_shrink)
+
+from .punica_base import PunicaWrapperBase
+
+
+@final
+class PunicaWrapperCPU(PunicaWrapperBase):
```

Comment:
The name `PunicaWrapperXPU` would be more appropriate, what do you think, @Isotr0py @Akshat-Tripathi

Reply:
I thought the XPU backend was for Intel GPUs?

Reply:
XPU backend is for Intel GPUs. I think we can just use CPU here, and inherit from CPU wrapper for XPU backend in the future.

Reply:
Make sense.